Readme.md

```diff
@@ -128,6 +128,14 @@ Here is the event repartition on ElasticSearch cluster over a 4h period
 
 ![](doc/images/es/event_type_repartition.png)
 
+### Common reader configuration
```

Comment:
Please just update the section HDFS URI standardization (not create a new one)
To clarify on which component it should be added

Reply:
done